src/clj/fluree/db/flake.cljc

```diff
@@ -31,7 +31,7 @@
 (def ^:const max-meta util/max-integer)
 
 
-(deftype Flake [s p o dt t op m]
+(deftype Flake [s p o dt t op m _meta]
```

Comment:
Tiny nitpick, but "_" is usually indicates arguments/fields that are never used. We don't use `_meta` in equality calculations, but we do/will use it in other ways. "-" usually indicates arguments/fields/functions that are "private" and shouldn't be used directly, so perhaps `-meta` better conveys the intention. 

I've been trying to think of a better name than "meta" to avoid confusion with the `m` field, but I haven't come up with anything yet. The best I can do is "clj-meta", which I'm not in love with. I don't think it's worth it to change the name of the `m` field at this point because that would be a HUGE change.

Reply:
I just changed it to `-clj-meta` as I like that suggestion the best. Take a look at let me know if it feels wrong.

Maybe we just need to find another word that starts with `m` and act like `.-m` was always for that. :)